src/util/format.cpp

```diff
@@ -97,4 +98,13 @@ std::string hex_format_buffer(const char *buf,
   return str;
 }
 
+std::string to_lower(const std::string &original)
```

Comment:
IIRC we have this elsewhere (maybe it's `to_upper`?)? Can you refactor and standardize so we don't have duplication, maybe this just means a stacked change to centralize the existing `to_upper`/`to_lower` in this file.

Reply:
There is no `to_upper`. And this change already consolidates this logic with the `Config` class (just not in a separate commit -  I can split it out if desired).